src/commonTest/kotlin/com/eiknat/lemmyclient/api/CommentTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.eiknat.lemmyclient.api
+
+import com.eiknat.lemmyclient.LemmyAPI
+import com.eiknat.lemmyclient.api.category.Category
+import com.eiknat.lemmyclient.api.comment.Comment
+import com.eiknat.lemmyclient.api.comment.form.GetCommentsForm
+import com.eiknat.lemmyclient.utils.MockClient
+import com.eiknat.lemmyclient.utils.executeTest
+import io.ktor.http.*
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.asserter
+
+class CommentTest {
+
+    @Test
+    fun `list comments`() {
+        executeTest {
+            MockClient(
+                httpMethod = HttpMethod.Get,
+                responseJson = commentJson,
+                endpoint = "/comment/list",
+                queryParams = "?type_=All&sort=Active&limit=2&community_id=2"
+            )
+
+            val form = GetCommentsForm(
+                type = ListingType.ALL,
+                sort = SortType.ACTIVE,
+                page = null,
+                limit = 2,
+                communityId = 2,
+                communityName = null,
+                auth = null
+            )
+
+            when (val res = Comment.list(form)) {
+                is APIResponse.Error -> { println("${res.statusCode} :: ${res.message}") ; asserter.fail("should not have failed") }
+                is APIResponse.Ok -> {
+                    assertEquals(2, res.data.comments.size, "response has two comments")
+                    val comment = res.data.comments.first()
+                    assertEquals(2, comment.communityId, "deserialized community id correctly")
+                }
+            }
+        }
+    }
+
+    private val commentJson = """
```

Comment:
I'd prefer to not have this in the code but I'm not sure where else it could go

Reply:
it's just for tests and they shouldn't link to anything real